SETUP.md

```diff
@@ -1,44 +1,80 @@
-- Scroll up to the top of the OAuth & Permissions page and click **Install App to Workspace**. You’ll be led through Slack’s OAuth UI, where you should allow your app to be installed to your development workspace.
-- Once you authorize the installation, you’ll land on the **OAuth & Permissions** page and see a **Bot User OAuth Access Token**.
-![](https://slack.dev/bolt-js/assets/bot-token.png)
+
+*   Navigate to the **OAuth & Permissions** on the left sidebar.
+    ![image](https://github.com/peculiaruc/peculiaruc.github.io/assets/35475543/07be85b8-7b28-4691-aab6-7d847b35c18e)
+
+*   Scroll down to the **Bot Token Scopes** section. Click **Add an OAuth Scope**. CHAOSS Slack bot currently has these \[OAuth Scopes]
+    ![image](https://github.com/peculiaruc/peculiaruc.github.io/assets/35475543/ed6d3dfb-4831-4b43-8b09-15cd7f3445b2) ![image](https://github.com/peculiaruc/peculiaruc.github.io/assets/35475543/ff3dda77-f205-4e95-8694-9e3d16594c36)! (https://docs.google.com/document/d/1NJd-nNKUNb3Q0lRb5cfmUU8kpRcYGjh-vPqpk4CCvic/edit#heading=h.v1ah7sirde2a) enabled, so enable those for full functionality of the bot. If you're adding a new function, you might need more permissions. See the set up for adding new functions below.
+
+A comprehensive list of permission scopes is available at [api.slack.com/scopes](https://api.slack.com/scopes).
+
+*   Scroll up to the top of the OAuth & Permissions page and click **Install App to Workspace**.
+    ![image](https://github.com/peculiaruc/peculiaruc.github.io/assets/35475543/550938ed-48f4-4296-9d73-0a3431f30182)
+    You’ll be led through Slack’s OAuth UI, where you should allow your app to be installed to your development workspace.
+    ![image](https://github.com/peculiaruc/peculiaruc.github.io/assets/35475543/be6977a7-ec87-4d42-b0fc-bcf09d04474c)
+    ![image](https://github.com/peculiaruc/peculiaruc.github.io/assets/35475543/6fc1643c-53a8-4b02-b3a7-bb84f2dfc046)
+    ![image](https://github.com/peculiaruc/peculiaruc.github.io/assets/35475543/3eab21ff-2dbc-4640-81de-245123ac265a)
+
+*   Once you authorize the installation, you will be led to  **OAuth & Permissions** page and see a **Bot User OAuth Access Token**.
+    ![](https://slack.dev/bolt-js/assets/bot-token.png)
+
 > Treat your token like a password and keep it safe. Your app uses it to post and retrieve information from Slack workspaces.
 
 # Setting up your project
-- Navigate to your project folder
-```
+
+*   Navigate to your project folder
+
+<!---->
+
 $ cd chaoss-slack-bot
```

Comment:
please put this in a code block, like it was before

Reply:
Resolved